packages/react/src/profile/infrastructure/ProfileCacheManager.ts

```diff
@@ -52,30 +46,6 @@ export class ProfileCacheManager implements IProfileCacheManager {
     });
   }
 
-  // TODO no longer need to accept a profile id, only authenticated profile can be used
-  updateProfile(id: string, updateFn: (current: Profile) => Profile): void {
```

Comment:
I removed this to promote optimistic implementations using type policies, it was not in use anyway

Reply:
😢